embassy-boot/boot/src/firmware_updater.rs

```diff
@@ -118,38 +119,37 @@ impl FirmwareUpdater {
         _state_and_dfu_flash: &mut F,
         _public_key: &[u8],
         _signature: &[u8],
-        _update_len: usize,
+        _update_len: u32,
```

Comment:
Unsure why we are changing this to a `u32`. Can you expand on why?

Reply:
I have reverted to usize. My plan is to change this to u32 throughout the bootloader, but that should probably be done in a separate pr. The reason for the u32 is to support 16 bit mcu's with flashes larger than 64k. Some of the msp430's have this.